### DISCUSSION TITLE
Discord Webhook Integration

### DISCUSSION THREAD

**Discussion by WhineyMonkey10** [pelican-dev/panel#49] [Ideas]
Either customisable discord webhooks for eggs that you can setup yourself depending on server actions, or pre-setup integration for specific eggs.

**Comment by wdhdev** [comment]
There is a plugin that exists for this for €5: https://www.sourcexchange.net/products/discord-notifications-for-pterodactyl

I've used it myself and it is quite good, I would recommend.

**Comment by WhineyMonkey10** [comment]
I still think built-in integration would be better. Why spend 5 euros when you can get it for free, especially since this is a new project so new feature like this are kind of what they're looking for.
I'll check it out though.

**Comment by lancepioch** [comment]
Please follow this issue: https://github.com/pelican-dev/panel/issues/9
